gtest/src/program.rs

```diff
@@ -248,6 +248,12 @@ impl<'a> Program<'a> {
     pub fn id(&self) -> ProgramId {
         self.id
     }
+
+    pub fn call_meta(&self, message: Option<IncomingMessage>, function_name: &str) -> Vec<u8> {
```

Comment:
public call meta should call "meta_state" function, no need to give custom variants at the moment

Reply:
and instead of the message, paste payload
copy-paste and reuse method as payload: T: AsRef<[u8]> -> Vec<u8> and second one with payload:  E: Encode -> D: Decode